index.html

```diff
@@ -50,41 +59,92 @@ <h2>Tonic</h2>
+            <h2 class="post">Tonic </h2>
+                <div class="frame">
+                    <img src="images/Frame2.png" alt="tabs">
+                </div>
+        <p class="content-text-px">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+        <div class="tags">
+            <div class="tags1">
+            <ul>
+                <li><a href="#" class="tag-item1">html</a></li>
+               <li><a href="#" class="tag-item1">css</a></li>
+                <li><a href="#" class="tag-item1">javascript</a></li>
+            </ul>
+        </div></div>
+        <div class="action">
+            <button type="button" class="button">See Project</button>
+        </div>
+        </div>
+        </div>
+         </div>
         <div>
             <div class="content">
-                <img src="images/snap2.png" alt="snapshoot-porfolio">
+                <div class="content2">
+                    <div class="block3">                                                                                                           
+                <img src="images/snap2.png" alt="snapshoot-porfolio" class="snap">
+                <img src="images/snappy3.png" alt="snapshoot-porfolio" class="snappy3">
+                </div>
+                <div class="block4">
                 <h2 class="post">Multi-Post <br>
                     Stories</h2>
                     <div class="frame">
                         <img src="images/Frame2.png" alt="tabs">
                     </div>
             <p class="content-text-px">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
             <div class="tags">
+                <div class="tags1">
                 <ul>
                     <li><a href="#" class="tag-item">html</a></li>
                     <li><a href="#" class="tag-item">css</a></li>
                     <li><a href="#" class="tag-item">javascript</a></li>
                 </ul>
+            </div></div>
+            <div class="action">
+                <button type="button" class="button">See Project</button>
+            </div>
             </div>
+            <div class="block5">
+                <h2 class="post">Multi-Post <br>
+                    Stories</h2>
+                    <div class="frame">
+                        <img src="images/frame4.png" alt="tabs">
+                    </div>
+            <p class="content-text-px">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <div class="tags1">
+                <ul>
+                    <li><a href="#" class="tag-item1">html</a></li>
+                    <li><a href="#" class="tag-item1">Ruby on Rails</a></li>
+                    <li><a href="#" class="tag-item1">css</a></li>
+                    <li><a href="#" class="tag-item1">javascript</a></li>
+                </ul>
```

Comment:
Kindly also use a flex box for showing the tags, since some of them are overflowing and some of them take too much space 😃

Your design:

![image](https://user-images.githubusercontent.com/73714615/188265832-b82450fd-658a-4006-aa94-3241590ff2d9.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188265794-69be2e6b-81f0-472b-9b62-ce880d061255.png)

Reply:
Kindly, I recommend you follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉